packages/react-reconciler/src/ReactFiberFlags.js

```diff
@@ -41,6 +41,7 @@ export const StoreConsistency = /*             */ 0b0000000000000100000000000000
 // possible, because we're about to run out of bits.
 export const ScheduleRetry = StoreConsistency;
 export const ShouldSuspendCommit = Visibility;
+export const DidDefer = ContentReset;
```

Comment:
I don't really know how we use ContentReset but I assume you've considered that whether this ever propagates through subtree flags and that the reuse can't inadvertantly mix flags?

Reply:
Yeah I picked one that wouldn’t conflict since we’re almost out of bits.